cmd/stolonctl/main.go

```diff
@@ -83,33 +84,37 @@ func init() {
 	stolonctlCmd.PersistentFlags().StringVar(&clusterConfig.Name, "cluster-name",
 		defaults.ClusterName, "Stolon cluster name")
 
-	bindFlagEnv(stolonctlCmd.PersistentFlags())
+	if err := bindFlagEnv(stolonctlCmd.PersistentFlags()); err != nil {
+		log.Warn(trace.DebugReport(err))
```

Comment:
Stack will not be readable if output like this. You can use field logging to avoid string escaping:
```go
log.WithError(err).Warn("Failed to bind environment to flags.") // ??
```
Besides, why is the error ignored - what does it mean if `bindFlagEnv` fails?

Reply:
I can hardly imagine how it can fail, but because linter fails on that line I just add error handling.
Otherwise, if it fails nothing really bad happens - command will use defaults.